src/pages/SignUp/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -1,27 +1,50 @@
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { InputLabel, MenuItem, SelectChangeEvent } from '@mui/material';
+import { FilterOptionsState, TextField } from '@mui/material';
+import Autocomplete, { createFilterOptions } from '@mui/material/Autocomplete';
```

Comment:
Maybe it could be better to merge autocomplete imports with others from the material ui?